datasets/winogrande/winogrande.py

```diff
@@ -0,0 +1,186 @@
+                    # These are the features of your dataset like images, labels ...
+                }
+            ),
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage="https://leaderboard.allenai.org/winogrande/submissions/get-started",
+            citation=_CITATION,
+            task_templates=[
+                QuestionAnsweringMultipleChoicesWithoutContext(
+                    question_column="question", answers_column="answers",
+                    options_column="options",
+                    task="question-answering-multiple-choices-without-context",
+                )
+            ],
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        # TODO(winogrande): Downloads the data and defines the splits
+        # dl_manager is a datasets.download.DownloadManager that can be used to
+        # download and extract URLs
+        dl_dir = dl_manager.download_and_extract(_URL)
+        data_dir = os.path.join(dl_dir, "winogrande_1.1")
+        return [
+            datalabs.SplitGenerator(
+                name=datalabs.Split.TRAIN,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, f"train_{self.config.data_size}.jsonl"),
+                    # 'labelpath': os.path.join(data_dir, 'train_{}-labels.lst'.format(self.config.data_size)),
+                    "split": "train",
+                },
+            ),
+            datalabs.SplitGenerator(
+                name=datalabs.Split.TEST,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"filepath": os.path.join(data_dir, "test.jsonl"), "split": "test"},
+            ),
+            datalabs.SplitGenerator(
+                name=datalabs.Split.VALIDATION,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "dev.jsonl"),
+                    # 'labelpath': os.path.join(data_dir, 'dev-labels.lst'),
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """Yields examples."""
+        # TODO(winogrande): Yields (key, example) tuples from the dataset
+        dict_map = {"A": 0, "B": 1, "C": 2, "D": 3, "E": 4, "F": 5, "G": 6, "H": 7, "I": 8, "J": 9, "K": 10}
+        id_sample = 0
+        with open(filepath, encoding="utf-8") as f:
+            for id_, row in enumerate(f):
+                data = json.loads(row)
+                id_sample += 1
+                options = [data["option1"], data["option2"]]
+                if split == "test":
```

Comment:
if one dataset isn't provided with the gold label, we can mark it in the `progress.md-comment`